### PR TITLE
feat: Reorganize python lambda section and add new example

### PR DESCRIPTION
**Pull request by wulfmann**
*Description of changes:*
Moves existing lambda exalmple to new lambda folder
Adds new lambda with s3 example
Adds new lambda with kinesis stream example

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by wulfmann** [comment]
If you are not looking to move examples into a lambda folder I can remove that part of the PR. I have a few other lambda examples I'm working on which is the only reason I did that.

**Comment by rhboyd** [comment]
I’m not a fan of service-specific directories. How would a new user know this example would be in the lambda folder and not in an S3 folder? I think the example name should make the use-case and/or services used clear

**Comment by wulfmann** [comment]
I do agree with your point there. My thinking was to follow the same convention as the S3 folder. I’ll update these examples for now and if the lambda examples become unwieldy we can revisit.

Thanks

**Comment by wulfmann** [comment]
I've pulled them out of a top-level lambda folder and left them independent.

**Comment by wulfmann** [comment]
Bump. Anything need to be updated on this to get it merged?

File: python/lambda-with-existing-s3-code/app.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            self, 'LambdaCodeBucket',
+            bucket_name='my-lambda-code-bucket'
+        )
+
+        lambdaFn = lambda_.Function(
+            self, 'Singleton',
+            handler='index.main',
+            code=lambda_.S3Code(
+                bucket=lambda_code_bucket,
+                key='my-lambdaa.py'
```
Comment (garnaat) [inline python/lambda-with-existing-s3-code/app.py]: Not a deal breaker but there is a typo there.

**Comment by wulfmann** [comment]
@garnaat thanks! updated

**Pull request merged**